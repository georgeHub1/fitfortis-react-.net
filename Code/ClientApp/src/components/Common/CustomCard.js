import React from 'react';
import { Card } from 'antd';

import styles from './styles.module.less';

const CustomCard = ({ title, extra, children, style = {} }) => {
  return (
    <Card
      style={{
        // backdropFilter: 'blur(116.886px)',
        background: 'rgba(255, 255, 255, 0.8)',
        ...style
      }}
      headStyle={{
        color: '#003459',
        borderBottom: 'none'
      }}
      className={`${styles.cardContainer} customCardContainer`}
      title={title}
      extra={extra}
    >
      {children}
    </Card>
  );
};

export default CustomCard;
