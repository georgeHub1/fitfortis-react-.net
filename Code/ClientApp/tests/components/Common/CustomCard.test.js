import React from 'react';
import renderer from 'react-test-renderer';

import CustomCard from '../../../src/components/Common/CustomCard.js';

const Container = CustomCard;

describe('CustomCard', () => {
  it('renders correctly', () => {
    const wrapper = renderer
      .create(<Container/>)
      .toJSON();

    expect(wrapper).toMatchSnapshot();
  });
});
