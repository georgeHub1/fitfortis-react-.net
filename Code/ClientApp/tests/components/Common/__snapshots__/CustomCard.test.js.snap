// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomCard renders correctly 1`] = `
<div
  className="ant-card undefined customCardContainer ant-card-bordered"
  style={
    Object {
      "background": "rgba(255, 255, 255, 0.8)",
    }
  }
>
  <div
    className="ant-card-body"
    style={Object {}}
  />
</div>
`;
